src/criteria/utils.js

```diff
@@ -144,3 +144,18 @@ export function setMatchingFunction( id, matchingFunction ) {
 	criteria._matched = {}; // Clear matched cache.
 	criteria.matchingFunction = matchingFunction;
 }
+
+/**
+ * Log debugging data if WP_DEBUG is set.
+ *
+ * @param {string} key  Key name for debug data.
+ * @param {any}    data Data to log.
+ */
+export function debug( key, data ) {
+	if ( ! newspack_popups_view.debug ) {
+		return;
+	}
+
+	window.newspack_popups_debug = window.newspack_popups_debug || {};
+	window.newspack_popups_debug[ key ] = data;
+}
```

Comment:
This seems like something that could live in a different utility js, like `src/prompts`. wdyt?

Reply:
Great idea. I restructured the files to something closer to this in e79cba4.